httpstan/cache.py

```diff
@@ -16,11 +15,23 @@
 logger = logging.getLogger("httpstan")
 
 
-def model_directory(model_name: str) -> str:
+def cache_directory() -> Path:
+    """Get httpstan cache path."""
+    return Path(appdirs.user_cache_dir("httpstan", version=httpstan.__version__))
+
+
+def model_directory(model_name: str) -> Path:
     """Get the path to a model's directory. Directory may not exist."""
-    cache_path = appdirs.user_cache_dir("httpstan", version=httpstan.__version__)
     model_id = model_name.split("/")[1]
-    return os.path.join(cache_path, "models", model_id)
+    return cache_directory() / "models" / model_id
+
+
+def fit_filepath(fit_name: str) -> Path:
```

Comment:
I'd stick with `fit_path` ("filepath" almost never occurs in the Python documentation; I think people have settled on "path" as the term)

Reply:
What do you think about the function? I tried to make it a bit more explicit, but I was not sure which one is cleaner solution

Split fit to model_id, fit_id and then call model_directory or just get cache path.

Reply:
I don't have any objection to it. It's a tiny bit odd that we have model_directory but not fit_directory. On the other hand, if we never refer to fit_directory directly, there's no need for a function returning it.